analysers/Analyser_Osmosis.py

```diff
@@ -105,6 +105,93 @@ class Analyser_Osmosis(Analyser):
 ;
 
 ANALYZE {0}.highway_ends;
+"""
+
+    sql_create_multipolygons = """
+DO $$
+DECLARE
+    mp RECORD;
+BEGIN
+    DROP TABLE IF EXISTS {0}.multipolygons;
+    CREATE UNLOGGED TABLE {0}.multipolygons (
+        LIKE relations INCLUDING ALL,
+        poly geometry(Geometry, 4326) NOT NULL,
+        poly_proj geometry(Geometry, {1}) NOT NULL,
+        is_valid boolean NOT NULL
+    );
+
+    FOR mp in (
+        SELECT
+            relations.*,
+            ST_LineMerge(ST_Collect(ways.linestring)) AS linestrings
+        FROM
+            relations
+            JOIN relation_members ON
+                relation_members.relation_id = relations.id AND
+                relation_members.member_type = 'W' AND
+                relation_members.member_role IN ('outer', 'inner')
+            JOIN ways ON
+                ways.id = relation_members.member_id
+        WHERE
+            relations.tags->'type' = 'multipolygon'
+        GROUP BY
+            relations.id
+    ) LOOP
+        BEGIN
+            IF ST_BuildArea(mp.linestrings) IS NOT NULL AND NOT ST_IsEmpty(ST_BuildArea(mp.linestrings)) THEN
+                WITH
+                unary AS (
+                    SELECT
+                        id, version, user_id, tstamp, changeset_id, tags,
```

Comment:
I'm not sure why `version, user_id, tstamp, changeset_id` need to be included? They I don't expect they'll be used in any analysers (and are also not present in `highways`, `buildings`, ...)?

Unless they're needed for `create_view_touched` for when an inner or outer way is changed (or a single node of an inner or outer way is moved, thereby causing i.e. the poly (and i.e. its area) to change, or ...)

Reply:
> I'm not sure why version, user_id, tstamp, changeset_id need to be included? I don't expect they'll be used in any analysers (and are also not present in highways, buildings, ...)?

To be removed.